src/pages/index.tsx

```diff
@@ -12,40 +13,45 @@ type Props = {
 }
 
 const AllCommunities = ({ communities }: Props): JSX.Element => {
-  const { account, library } = useWeb3React()
-
-  const isConnected = typeof account === "string" && !!library
+  const refYours = useRef<HTMLDivElement>(null)
+  const refAccess = useRef<HTMLDivElement>(null)
+  const refOther = useRef<HTMLDivElement>(null)
 
```

Comment:
I'd like to have a comment here briefly explaining the Portal method we're using (including the why) since it's not an ordinary thing to do